test/unit-tests/MultiBridgeMessageReceiver.t.sol

```diff
@@ -262,7 +277,7 @@ contract MultiBridgeMessageReceiverTest is Setup {
         vm.startPrank(wormholeAdapterAddr);
         receiver.receiveMessage(message);
 
-        receiver.scheduleMessageExecution(msgId);
+        receiver.scheduleMessageExecution(msgId, MessageLibrary.extractExecutionParams(message));
```

Comment:
nit: Might be worth considering the following minor refactoring for readability in this instance and other similar instances: 
- add `using MessageLibrary for MessageLibrary.Message;` to the contract
- replacing `MessageLibrary.extractExecutionParams(message)` -> `message.extractExecutionParams()`

Reply:
Done